nifi-ngsi-bundle/nifi-ngsi-processors/src/test/java/org/apache/nifi/processors/ngsi/TestNGSIToPostgreSQL.java

```diff
@@ -871,11 +871,13 @@ public void testGetNewListOfFields() throws Exception {
         Map<String, POSTGRESQL_COLUMN_TYPES> listOfFields = new TreeMap<>();
         listOfFields.put("temperature", POSTGRESQL_COLUMN_TYPES.TEXT);
 
-        Map<String, POSTGRESQL_COLUMN_TYPES> newListOfFields = backend.getUpdatedListOfTypedFields(resultSetMock, listOfFields);
+        assertEquals(POSTGRESQL_COLUMN_TYPES.TEXT, listOfFields.get("temperature"));
```

Comment:
just one example here: you add `temperature` on line 872 and you check two lines below that it is in the map, so you are testing Java maps?

Reply:
Ok I see I change that